### PR TITLE
16,383 is the max length of tokens in characters

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: modules/ROOT/pages/database-internals/store-formats.adoc [hunk 1]
```diff
@@ -24,7 +24,8 @@ Block format means a few pages need to be loaded to serve a query, i.e. fewer pa
 * *Property access:* Properties are stored in blocks with their nodes and relationships drastically reducing the amount of pointer chasing required to access properties.
 * *Entity limits:* Able to run graphs at large scales.
 Supports the highest limits at the time of writing.
-See <<block-format-limits, Block format entity limits>> for details.
+See <<block-format-limits, Block format entity limits>> for details. +
+label:new[Introduced in 5.25] The max length of token names (i.e., label, property key, and relationship type names) is 16,383 characters.
```
Comment (Roxling) [inline modules/ROOT/pages/database-internals/store-formats.adoc:28]: Should we link to or mention that 16,383 characters is a GQL identifier max length?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1902.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1902.surge.sh/operations-manual/5/database-internals/store-formats)
- [Import](https://neo4j-docs-operations-1902.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1902.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1902.surge.sh/operations-manual/5/database-internals/store-formats)
- [Import](https://neo4j-docs-operations-1902.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by renetapopova** [comment]
The failing test is note related to this PR.

**Review by Roxling — APPROVED** [review]

**Review by EmelieBondesson — APPROVED** [review]

**Pull request merged**